lib/api/LogManagerImpl.cpp

```diff
@@ -402,7 +402,7 @@ namespace MAT_NS_BEGIN
             m_httpClient = nullptr;
             m_taskDispatcher = nullptr;
             m_dataViewer = nullptr;
-            m_dataInspector = nullptr;
+            m_dataInspectors.clear();
```

Comment:
Clear doesn't guarantee that the underlying buffer is released. Prefer std::vector<T>()::swap(m_dataInspectors), as this guarantees that the underlying buffer is released.
See: https://www.cplusplus.com/reference/vector/vector/clear/

Reply:
Actually, shouldn't this just call ClearDataInspectors()?

Reply:
changed to call `ClearDataInspectors`